build/pipelines/release.yml

```diff
@@ -84,671 +98,113 @@ variables:
-      env:
-        LIB: $(Build.SourcesDirectory)
-        ArtifactServices_Symbol_AccountName: microsoftpublicsymbols
-        ArtifactServices_Symbol_PAT: $(ADO_microsoftpublicsymbols_PAT)
-
-
-- ${{ if eq(parameters.buildTerminalVPack, true) }}:
-  - job: VPack
-    displayName: Create Windows vPack
-    dependsOn: BundleAndSign
-    steps:
-    - checkout: self
-      clean: true
-      fetchDepth: 1
-      submodules: true
-    - task: PkgESSetupBuild@12
-      displayName: Package ES - Setup Build
-    - task: DownloadBuildArtifacts@1
-      displayName: Download Build Artifacts
-      inputs:
-        artifactName: appxbundle-signed
-        extractTars: false
-    - task: PowerShell@2
-      displayName: Rename and stage packages for vpack
-      inputs:
-        targetType: inline
-        script: >-
-          # Rename to known/fixed name for Windows build system
-
-          Get-ChildItem Microsoft.WindowsTerminal_*.msixbundle | Rename-Item -NewName { 'Microsoft.WindowsTerminal_8wekyb3d8bbwe.msixbundle' }
-
-
-          # Create vpack directory and place item inside
 
-          mkdir WindowsTerminal.app
+stages:
+  - stage: Build
+    displayName: Build
+    dependsOn: []
+    jobs:
+      - template: ./templates-v2/job-build-project.yml
+        parameters:
+          pool:
+            name: SHINE-INT-L # Run the compilation on the large agent pool, rather than the default small one.
+            demands: ImageOverride -equals SHINE-VS17-Latest
+          branding: ${{ parameters.branding }}
+          buildTerminal: ${{ parameters.buildTerminal }}
+          buildConPTY: ${{ parameters.buildConPTY }}
+          buildWPF: ${{ parameters.buildWPF }}
+          pgoBuildMode: ${{ parameters.pgoBuildMode }}
+          buildConfigurations: ${{ parameters.buildConfigurations }}
+          buildPlatforms: ${{ parameters.buildPlatforms }}
+          generateSbom: ${{ parameters.generateSbom }}
+          codeSign: ${{ parameters.codeSign }}
+          beforeBuildSteps: # Right before we build, lay down the universal package and localizations
+            - task: PkgESSetupBuild@12
+              displayName: Package ES - Setup Build
+              inputs:
+                disableOutputRedirect: true
+
+            - task: UniversalPackages@0
+              displayName: Download terminal-internal Universal Package
+              inputs:
+                feedListDownload: 2b3f8893-a6e8-411f-b197-a9e05576da48
+                packageListDownload: e82d490c-af86-4733-9dc4-07b772033204
+                versionListDownload: ${{ parameters.terminalInternalPackageVersion }}
+
+            - template: ./templates-v2/steps-fetch-and-prepare-localizations.yml
+              parameters:
+                includePseudoLoc: true
+
+      - ${{ if eq(parameters.buildWPF, true) }}:
+        # Add an Any CPU build flavor for the WPF control bits

```

Comment:
This one is interesting. It takes the normal `build-project` job and runs it AGAIN with a different set of parameters. The most important one is buildPlatforms = `[Any CPU]`